frontend/src/utilities/routeUtils.ts

```diff
@@ -0,0 +1,67 @@
+export const routes = {
+  '/': 'Enabled',
+  '/clusterSettings': 'Settings',
+  '/enabled': 'Enabled',
+  '/explore': 'Explore',
+  '/groupSettings': 'Group Settings',
+  '/notebook': 'Notebook',
+  '/notebook/:namespace': (value: string): string => value,
```

Comment:
@andrewballantyne Not sure what we said dynamic links should return.   We have them for notebooks as well as dsg.  Thanks.

Reply:
I believe we want to use annotations now after we talked today.

Reply:
How do you contextually know how to render this? I was thinking maybe this component was on each page we want -- but this is on every page because it's done at the top of the app 🤔 

`:namespace` might be easy to do if we can figure out a way to generically talk about the Project object (or even maybe fetching it live or something...). But what about custom items 🤔 `:foo` -- how do we figure that out generically?

I'm going to keep thinking on it -- this would be a neat feature to have but I'm not sure how we can achieve the generic nature of it.